linode_api4/groups/linode.py

```diff
@@ -311,13 +349,6 @@ def instance_create(
             ret_pass = Instance.generate_root_password()
             kwargs["root_pass"] = ret_pass
 
-        interfaces = kwargs.get("interfaces", None)
-        if interfaces is not None and isinstance(interfaces, Iterable):
-            kwargs["interfaces"] = [
-                i._serialize() if isinstance(i, ConfigInterface) else i
-                for i in interfaces
-            ]
```

Comment:
No longer necessary due to `_flatten_request_body_recursive(drop_null_keys(params))` below